garage/envs/wrappers/gray_scale.py

```diff
@@ -0,0 +1,35 @@
+"""Grayscale wrapper for gym.Env."""
+import gym
+from gym.spaces import Box
+import numpy as np
+from skimage import color
+
+
+class GrayScale(gym.Wrapper):
+    """Converting frames to grayscale."""
```

Comment:
Calling env.render() on the wrapped environment still render colour image. Probably it's their way of doing it. 

Env.render(mode='rgb_array') still gives the colour image, but it returns the raw-pixel as array instead of displaying it.

But yes, if the user wants to see gray image, we have to override render() and display the processed image.

Reply:
what is the gym API contract? isn't it

* `mode == "human"` --> show a window, return none
* `mode == "rgb_array"` --> show nothing, return pixels

If I want to get pixels and show a window, I should be able to call:
```python
env.render(mode="human")  # shows a window
pixels = env.render(mode="rgb_array")  # gives me the pixels
```